corehq/apps/app_manager/templates/app_manager/v1/source_files.html

```diff
@@ -57,9 +57,15 @@
     {% compress js %}
         <script src="{% static 'hqwebapp/js/urllib.js' %}"></script>
         <script src="{% static 'hqwebapp/js/initial_page_data.js' %}"></script>
+        <script src="{% static 'Caret.js/dist/jquery.caret.js' %}"></script>
+        <script src="{% static 'At.js/dist/js/jquery.atwho.min.js' %}"></script>
```

Comment:
No strong preference between the two. I used atwho here out of laziness (I'm more familiar with it that with select2)...but looked at it again today, and select2 makes the code a lot shorter. Will update.